github.com.hashicorp.go.kms.wrapping.v2.types.proto

```diff
@@ -119,3 +133,33 @@ enum HmacType {
 	SHA384 				= 3;
 	SHA512 				= 4;
 }
+ 
+// KeyType defines the key's type
+  enum KeyType {
```

Comment:
Can you normalize the whitespacing here? (Also in HmacType above)

Reply:
done.